users.go

```diff
@@ -292,6 +292,13 @@ func GetUsersOptionPresence(n bool) GetUsersOption {
 	}
 }
 
+// GetUsersOptionTeamID include team_id
+func GetUsersOptionTeamID(team_id string) GetUsersOption {
```

Comment:
Prefer to use camel case in Go projects.

Reply:
Cool, have updated to camelCase